packages/react-native-gesture-handler/jestSetup.js

```diff
@@ -7,7 +7,7 @@ jest.mock('./lib/commonjs/RNGestureHandlerModule', () =>
   require('./lib/commonjs/mocks/mocks')
```

Comment:
1. Can this be renamed from `mocks`?
2. I don't think it should contain anything more than the module methods. The old gesture handlers, gesture-wrapped components, and host detector should be separate and mocked to the correct files, no?

I think the current setup dates back to the time when all of Gesture Handler was a single file, so it was correct to mock everything to a single file.